src/webhooks/kafka-control-plane/kafka-control-plane.test.ts

```diff
@@ -33,7 +33,7 @@ describe('kafka-control-plane webhook', function () {
       method: 'POST',
       url: '/metrics/kafka-control-plane/webhook',
       headers: {
-        'x-kafka-control-plane-signature': 'invalid',
+        'x-infra-event-notifier-signature': 'invalid',
```

Comment:
I think in infra-event-notifier I have it using the old kafka header. Since the endpoint is called kafka-control-plane, should we keep the old header name?